llvm/test/CodeGen/AIE/GlobalISel/eliminate-duplicate-phi.mir

```diff
@@ -232,3 +232,51 @@ body:             |
     %5:_(p0) = G_PTR_ADD %3:_, %4:_(s20)
     G_BR %bb.4
 ...
+
+
+---
+name:            test_two_PHI_nobranch
+legalized:       false
+tracksRegLiveness: true
+body:             |
+  ; CHECK-LABEL: name: test_two_PHI_nobranch
+  ; CHECK: bb.0:
+  ; CHECK-NEXT:   successors: %bb.1(0x40000000), %bb.2(0x40000000)
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   [[C:%[0-9]+]]:_(s32) = G_CONSTANT i32 0
+  ; CHECK-NEXT:   [[C1:%[0-9]+]]:_(p0) = G_CONSTANT i20 1
+  ; CHECK-NEXT:   [[C2:%[0-9]+]]:_(p0) = G_CONSTANT i20 2
+  ; CHECK-NEXT:   G_BRCOND [[C]](s32), %bb.1
+  ; CHECK-NEXT:   G_BR %bb.2
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT: bb.1:
+  ; CHECK-NEXT:   successors: %bb.3(0x80000000)
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   G_BR %bb.3
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT: bb.2:
+  ; CHECK-NEXT:   successors: %bb.3(0x80000000)
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   G_BR %bb.3
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT: bb.3:
+  ; CHECK-NEXT:   [[C2:%[0-9]+]]:_(p0) = G_PHI [[C1]](p0), %bb.1, [[C2]](p0), %bb.2
+  ; CHECK-NEXT:   [[PHI:%[0-9]+]]:_(p0) = G_PHI [[C1]](p0), %bb.1, [[C2]](p0), %bb.2
+  bb.0:
+    %1:_(s32) = G_CONSTANT i32 0
+    %2:_(p0) = G_CONSTANT i20 1
+    %3:_(p0) = G_CONSTANT i20 2
+    G_BRCOND %1:_(s32), %bb.1
+    G_BR %bb.2
+
+  bb.1:
+    G_BR %bb.3
+
+  bb.2:
+    G_BR %bb.3
+
+  bb.3:
+    %3:_(p0) = G_PHI %2:_(p0), %bb.1, %3:_(p0), %bb.2
+    %4:_(p0) = G_PHI %2:_(p0), %bb.1, %3:_(p0), %bb.2
```

Comment:
This is just triggering the crash. These are equivalent phis, and the test can be strengthened by using %4 in a downstream block.